### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by phantom-terminal**

### CODE REVIEW

**Review by alexandra-halytska — APPROVED** [review]

File: tests/test_main.py [hunk 1]
```diff
@@ -1 +1,22 @@
...
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_prediction_is_bearish(mocked_prediction_rate):
+    mocked_prediction_rate.return_value = 94.99
+    assert cryptocurrency_action(100) == "Sell all your cryptocurrency"
+
+
+@mock.patch("app.main.get_exchange_rate_prediction")
+def test_prediction_is_in_range_more_or_less_5_percent(mocked_prediction_rate):
+    mocked_prediction_rate.return_value = 10
```
Comment (MasterpieceElbow) [inline tests/test_main.py:19]: Add boundary conditions, you don't check when `get_exchange_rate_prediction` return value equals exactly 1.05 or 0.95 * value passed in `cryptocurrency_action`.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]
